modules/pricer/src/main/java/com/opengamma/strata/pricer/impl/option/SabrExtrapolationRightFunction.java

```diff
@@ -504,7 +504,7 @@ public double getTimeToExpiry() {
           break;
         default:
           param = sabrData.getNu();
-          paramShift = param * shift; // Relative shift to cope with difference in order of magnitude.
+          paramShift = shift; // rho can be zero, so relative shift is not possible.
```

Comment:
The comment should be "nu can be zero, ...". 
Seems we need changes to paramShift for the alpha parameter: any value is allowed in SabrFormulaData, whereas zero is the boundary in SabrModelFitter.